### ISSUE TITLE
Consider moving the project to gitlab.freedesktop.org

### ISSUE THREAD

**Issue by ddevault** [xkbcommon/libxkbcommon#259]

**Comment by bluetech** [comment]
Makes sense. We actually used to be on the old freedesktop and I prefer it over a proprietary service in principle.

Here's what I think would be needed:

- All contributors should agree (@fooishbar and @whot).
- Be able to create an "organization" i.e. the repo should not be under an individual user.
- Issue / PR history should be migrated (also https://github.com/xkbcommon/website). 
- Should allow requiring 2FA for pushing/merging/releasing etc (everything except issues/PRs).
- CI jobs should be migrated.
- Mirroring the git data back to this repo (GitHub), since many places use it.

I should clarify that I do not intend to do this myself.

**Comment by whot** [comment]
ACK from me, provided someone else does the work...

> Be able to create an "organization" i.e. the repo should not be under an individual user.

That's easy enough or if you want to we could shove it in with the [xkeyboard-config](https://gitlab.freedesktop.org/xkeyboard-config) group

> CI jobs should be migrated.

converting the linux job to ci-templates to run on gitlab would be a 15min job for me. No idea how to do  the macos and windows jobs though. Mesa has some windows jobs in its CI which we could presumably use as inspiration.


I don't know about issue migration, 2FA enforcement (beyond requiring all users to do so) and how to set up mirroring to github (I'm sure that's just a google away though).

**Comment by fooishbar** [comment]
I'm happy for it to be moved, but don't have the time to tackle stuff right now.

We can enforce 2FA, issues are definitely migrated when you import and I'm pretty sure PRs are as well.

We don't have a way to migrate macOS CI. Linux and Windows are easy though.

**Comment by ddevault** [comment]
Bump?
